build_bundle.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/python
+import json
+import ConfigParser
+import sys
+import os
+import tempfile
+import glob
+import subprocess
+import shutil
+import time
+
+#1) metadata
+#2) rootfs
+#3) binary
+
+#CONFIG:
+#    component : DomD item domd
+CONFIG_FILE_NAME = "xt-bundle-aos.cfg"
+
+BUNDLE_CONFIG_SECTION = "bundle_config"
+TOP_TEMPLATE = "top_template"
+UM_TEMPLATE = "um_template"
+COMPONENTS = "components"
+OPT_DEPLOY_DIR = "bundle_deploy_dir"
+
+class BundleConfig(object):
+    def __init__(self):
+        print("bundle_config read")
+        self.__config = ConfigParser.ConfigParser()
+        self.__config.read(CONFIG_FILE_NAME)
+        #TODO get cmdline param 
+
+    def get_top_template(self):
+        print("get_top_template")
+        return self.__config.get(BUNDLE_CONFIG_SECTION, TOP_TEMPLATE, 1)
+
+    def opt_exist(self, section, option):
+        return self.__config.has_option(section, option)
+
+    def get_option(self, section, option):
+        return self.__config.get(section, option, 1)
+
+OPT_ENABLE = "enabled"
+OPT_TYPE = "type"
+OPT_ITEM_TEMPLATE = "item_template"
+OPT_DEPENDENCIES = "dependencies"
+
+SECT_PATH = "path"
+OPT_BASE = "workspace_base_dir"
+
+RUNTIME_DEPS = "runtimeDependencies"
+class BundleProcessor(object):
```

Comment:
Add space before class, do you use any standard python formatter? yapf, for example.